Makefile

```diff
@@ -431,7 +431,7 @@ endif
 # be JIT-compiled by the updated driver from the included PTX.
 ifeq ($(USE_CUDA), 1)
 ifeq ($(CUDA_ARCH),)
-	KNOWN_CUDA_ARCHS := 30 35 50 52 60 61 70 75
+	KNOWN_CUDA_ARCHS := 30 35 50 52 60 61 70 75 80
```

Comment:
we are removing makefile in #18721

Reply:
I will revert the commit that added '80' since the Makefile will soon be removed.